src/i965_drv_video.c

```diff
@@ -1043,18 +1043,23 @@ i965_GetConfigAttributes(VADriverContextP ctx,
                     profile == VAProfileH264Main ||
                     profile == VAProfileH264High) {
 
-                    VAConfigAttribValEncROI *roi_config =
-                        (VAConfigAttribValEncROI *)&(attrib_list[i].value);
+                    if (IS_GEN9(i965->intel.device_info))
```

Comment:
Agree with what Scott said.
Please only disable it for non-supported platforms/